scripts/clang_format.sh

```diff
@@ -39,7 +39,7 @@ function check_clang_format() {
 }
 
 function clang_format_run() {
-  ${CLANG_FORMAT_CMD} -i -style=Google "$@"
```

Comment:
This overrides the settings in `.clang-format`

Reply:
> This overrides the settings in `.clang-format`

Better note this in this file.